components/x-audio/src/redux/swipe-down-to-minimise.js

```diff
@@ -0,0 +1,31 @@
+const EXPANDED_PLAYER_HEIGHT = 360;
+const DISPLAY_EXPANDED_THRESHOLD = EXPANDED_PLAYER_HEIGHT / 2;
+
+function setPositionY (target, posY) {
+	target.style.transform = `translate3d(0, ${posY}px, 0)`;
+}
+
+export default (event, playerActions) => {
+
+	const direction = event.offsetDirection;
```

Comment:
if we only listen to `DIRECTION_DOWN` then we presumably don't need these checks

Reply:
👍